bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,15 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  user_defined_constant = c('constant', (upper + (P'_' * upper)) * (upper + '_')^0) * -#(alpha + digit)
 
   dunder_identifier = c 'special', (P'__' * (alpha^1 * (alpha + digit)^0) * P'__')
 
   identifier = c 'identifier', name
   fdecl = c('keyword', 'def') * c('whitespace', space^1) * c('fdecl', name)
   classdef = c('keyword', 'class') * c('whitespace', space^1) * c('type_def', name)
+  self = c 'member', word {'self'}
+  member = self * c('operator', '.') * c('member', name)
```

Comment:
Sorry I left this for the last minute, but Python syntax technically allows whitespace in before and after the `.`, like `self    .  xyz`.